tensorbay/opendataset/BioIDFace/catalog.json

```diff
@@ -0,0 +1,37 @@
+{
+    "KEYPOINTS2D": {
+        "keypoints": [
+            {
+                "number": 2,
+                "names": ["L", "R"],
+                "parentCategories": ["Eye Position"]
+            },
+            {
+                "number": 20,
+                "names": [
+                    "right eye pupil",
+                    "left eye pupil",
```

Comment:
There are `right eye pupil` and `left eye pupil` in the second group of keypoints here.
Are they the same of the `L` and `R` in the first group?